public/app/features/dashboard/utils/getPanelChromeProps.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+
+import { LinkModel, PanelData, PanelPlugin, renderMarkdown } from '@grafana/data';
+import { getTemplateSrv, locationService, reportInteraction } from '@grafana/runtime';
+import { PanelPadding } from '@grafana/ui';
+import { InspectTab } from 'app/features/inspector/types';
+import { getPanelLinksSupplier } from 'app/features/panel/panellinks/linkSuppliers';
+import { plugin } from 'app/plugins/panel/alertGroups/module';
```

Comment:
@axelavargas I discovered a bug, the panel padding is not working / correctly set for panels that have the "noPadding" option set (Stat panel, old angular table panel). 

The problem is that your importing plugin from here instead of accessing it from props :)

Reply:
oh! will fix it :)